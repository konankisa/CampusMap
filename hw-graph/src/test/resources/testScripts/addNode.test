# Creates an empty graph and adds a node to it

# Create a graph
CreateGraph graph

# Add a node
AddNode graph node