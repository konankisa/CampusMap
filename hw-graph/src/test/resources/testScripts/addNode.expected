# Creates an empty graph and adds a node to it

# Create a graph
created graph graph

# Add a node
added node node to graph